plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/CacheMediatorDeserializer.java

```diff
@@ -59,26 +60,27 @@ public CacheMediator createNode(IGraphicalEditPart part, AbstractMediator object
 			if (mediator.isCollector()) {
 				executeSetValueCommand(CACHE_MEDIATOR__CACHE_ACTION, CacheAction.COLLECTOR);
 			} else {
-				String[] methods = mediator.getHTTPMethodsToCache();
-				if (methods.length != 0 && !methods[0].isEmpty()) {
-					StringBuilder method = new StringBuilder();
-					for (int i = 0; i < methods.length - 1; i++) {
-						method.append(methods[i]).append(", ");
+				if (CachingConstants.HTTP_PROTOCOL_TYPE.equals(mediator.getProtocolType())) {
```

Comment:
Shall we check for spacing after and inside else?

Reply:
Formatting issues are fixed in : #272